template.rb

```diff
@@ -4,7 +4,7 @@
 
 ruby_version = '2.3.0'
 insert_into_file "Gemfile", "ruby '#{ruby_version}'", after: "source 'https://rubygems.org'\n"
-
+insert_into_file "Gemfile", "  gem 'pry-rails'\n", after: "gem 'byebug'\n"
```

Comment:
Instead, you could use `gem 'pry-rails'` to directly place the gem into the file.